app/README.md

```diff
@@ -13,7 +13,7 @@
    and rebuild as needed.
 5. Run `npm run SHORT_NAME` to run a development server serving one of the three
    portals on port 5000.
-6. Run `./scripts/run-development-apis.sh` to run both APIs (main and reporting)
+6. Run `./scripts/run-development-apis.sh` to run Montagu
```

Comment:
I think this should be run-development-dependencies now?

Reply:
good spot!